### PR TITLE
Footer changes #3

### LINKED ISSUE #3: Footer

### PR DESCRIPTION
**Pull request by abhishekshar-adobe**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #3 

Test URLs:
- Before: https://main--airindia--hlxsites.hlx.live/
- After: https://footer--airindia--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/separator/separator.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.separator{
+    background-color: var(--color-white);
+    border: 0.6px dashed var(--ai-color-dark-header);
+    color: var(--color-white);
+    margin: 20px 0;
+    opacity: .25;
+}
+
+@media (width <= 767px) {
```
Comment (sdmcraft) [inline blocks/separator/separator.css:9]: Let's keep mobile styles as default and specify desktop styles via media queries. Here's its the opposite.

**Review by sdmcraft — COMMENTED** [review]
Looks like missing left margins
image

**Review by sdmcraft — COMMENTED** [review]
Layout seems to break in the range on 992px to 1200px width

image

File: blocks/separator/separator.css (hunk 1 above)
Comment (sselvara) [reply blocks/separator/separator.css:9]: @abhishekshar-adobe , footer is breaking in safari as Media Queries: Range Syntax is not supported in safari <16.3, https://caniuse.com/css-media-range-syntax.

Use mind-width syntax to fix the same pls.
image

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -135,7 +142,7 @@ footer .footer .footer-top-item p a{
   transform: rotate(270deg);
 }
 
-@media (width >= 768px) {
+  @media (min-width: 768px) {
```
Comment (sdmcraft) [inline blocks/footer/footer.css:145]: Extra space

**Review by sdmcraft — APPROVED** [review]
Looks good overall. 1 comment, a section in is not centrally aligned.
image

**Pull request merged**